bndtools.core/bnd.bnd

```diff
@@ -13,6 +13,8 @@ Import-Package: \
  ${eclipse.importpackage},\
  *
 
+Export-Package: org.osgi.service.metatype.annotations 
```

Comment:
Why are we exporting a package of class retention annotations? Who would be importing them?